app/enterprise.go

```diff
@@ -120,6 +120,9 @@ func RegisterNotificationInterface(f func(*App) einterfaces.NotificationInterfac
 }
 
 func (s *Server) initEnterprise() {
+	if metricsInterface != nil {
```

Comment:
I was looking at how can we change this to avoid being order-dependent but is good by now